lib/frrscript.h

```diff
@@ -149,53 +177,81 @@ const struct prefix * : lua_decode_noop                         \
 )(L, -1, value)
 
 /*
- * Call script.
+ * Call Lua function state (abstraction for a single Lua function)
  *
- * fs
- *    The script to call; this is obtained from frrscript_load().
+ * lfs
+ *    The Lua function to call; this should have been loaded in by
+ * frrscript_load(). nargs Number of arguments the function accepts
```

Comment:
indent to line up with `T` please

Reply:
updated in https://github.com/FRRouting/frr/pull/8982/commits/08bf2b41b82f3bc46c052044a4630a8534b819e9